components/admin_console/__snapshots__/schema_admin_settings.test.jsx.snap

```diff
@@ -326,7 +326,7 @@ exports[`components/admin_console/SchemaAdminSettings should match snapshot with
                 },
                 Object {
                   "order": 8,
-                  "text": "Polski (Alpha)",
+                  "text": "Polski",
```

Comment:
Line 455 https://github.com/mattermost/mattermost-webapp/blob/4a5110ebbf885076b031b8b913bcd19c01907cf0/components/admin_console/__snapshots__/schema_admin_settings.test.jsx.snap#L455 can be updated too.

Reply:
Thank you!